cmd/get/execution.go

```diff
@@ -22,24 +22,27 @@ Retrieves all the executions within project and domain.(execution,executions can
  bin/flytectl get execution -p flytesnacks -d development
 
 Retrieves execution by name within project and domain.
-
 ::
 
  bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r
 
-Retrieves execution by filters
+Retrieves all the tasks with filters.
 ::
 
- Not yet implemented
+ bin/flytectl get execution -p flytesnacks -d development oeh94k9r2r --field-selector="execution.name=oeh94k9r2r" 
```

Comment:
Add one example with composite filter and also what does it mean to have 
as argument oeh94k9r2r as well as field selector .
We should probably avoid accepting argument if field selector is added

Reply:
+1 it's a little confusing to have the filter repeat a value already specified :)

Reply:
Yes, it's a documentation issue. I fixed it